seq/detail/masterConfig.cpp

```diff
@@ -144,56 +144,61 @@ bool MasterConfig::handleEvent( const eq::ConfigEvent* event )
-
-      case eq::Event::KEY_PRESS:
-      case eq::Event::KEY_RELEASE:
-          if( Config::handleEvent( event ))
-          {
-              _redraw = true;
-              LBVERB << "Redraw: requested by eq::Config" << std::endl;
-          }
-          // no break;
-      case eq::Event::CHANNEL_POINTER_BUTTON_RELEASE:
-      case eq::Event::CHANNEL_POINTER_MOTION:
-      case eq::Event::CHANNEL_POINTER_WHEEL:
-      case eq::Event::MAGELLAN_AXIS:
-      {
-          if( _currentViewID == 0 )
-              return false;
-
-          View* view = static_cast<View*>( find<eq::View>( _currentViewID ));
-          if( view->handleEvent( event ))
-          {
-              _redraw = true;
-              LBVERB << "Redraw: requested by view event handler" << std::endl;
-          }
-          return true;
-      }
-
-      case eq::Event::STATISTIC:
-      {
-          Config::handleEvent( event );
-          if( event->data.statistic.type != eq::Statistic::CONFIG_FINISH_FRAME )
-              return false;
-
-          ViewUpdateVisitor viewUpdate( _redraw );
-          accept( viewUpdate );
-          return _redraw;
-      }
-
-      case eq::Event::WINDOW_EXPOSE:
-      case eq::Event::WINDOW_RESIZE:
-      case eq::Event::WINDOW_CLOSE:
-      case eq::Event::VIEW_RESIZE:
-          _redraw = true;
-          LBVERB << "Redraw: window change" << std::endl;
-          break;
+    case eq::Event::CHANNEL_POINTER_BUTTON_PRESS:
+        _currentViewID = event->data.context.view.identifier;
+        return true;
 
-      default:
-          break;
+    case eq::Event::KEY_PRESS:
+    case eq::Event::KEY_RELEASE:
+        if( Config::handleEvent( event ))
+        {
+            _redraw = true;
+            LBVERB << "Redraw: requested by eq::Config" << std::endl;
+        }
+        // no break;
+    case eq::Event::CHANNEL_POINTER_BUTTON_RELEASE:
+    case eq::Event::CHANNEL_POINTER_MOTION:
+    case eq::Event::CHANNEL_POINTER_WHEEL:
+    case eq::Event::MAGELLAN_AXIS:
+    {
+        if( _currentViewID == 0 )
+            return false;
+
+        View* view = static_cast<View*>( find<eq::View>( _currentViewID ));
+        if( view->handleEvent( event ))
+        {
+            _redraw = true;
+            LBVERB << "Redraw: requested by view event handler" << std::endl;
+        }
+        return true;
+    }
+
+    case eq::Event::STATISTIC:
+    {
+        Config::handleEvent( event );
+        if( event->data.statistic.type != eq::Statistic::CONFIG_FINISH_FRAME )
+            return false;
+
+        ViewUpdateVisitor viewUpdate( _redraw );
+        accept( viewUpdate );
+        return _redraw;
+    }
+
+    case eq::Event::WINDOW_EXPOSE:
+    case eq::Event::WINDOW_RESIZE:
+    case eq::Event::WINDOW_CLOSE:
+    case eq::Event::VIEW_RESIZE:
+        _redraw = true;
+        LBVERB << "Redraw: window change" << std::endl;
+        break;
+
+    case EVENT_REDRAW:
+        _redraw = true;
+        LBINFO << "Redraw request" << std::endl;
```

Comment:
Oops. Happened when realizing I was in the wrong event handling ifdef path. When is the event cleanup merged? *poke*